{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"conflictException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/ConflictException","preciseIdentifier":"s:10AWSAppflow17ConflictExceptionV","text":"ConflictException"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsappflow\/deleteconnectorprofileoutputerror\/conflictexception(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/DeleteConnectorProfileOutputError\/conflictException(_:)","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"conflictException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ConflictException","preciseIdentifier":"s:10AWSAppflow17ConflictExceptionV"},{"kind":"text","text":")"}],"title":"DeleteConnectorProfileOutputError.conflictException(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:10AWSAppflow33DeleteConnectorProfileOutputErrorO17conflictExceptionyAcA08ConflictH0VcACmF","modules":[{"name":"AWSAppflow"}]},"hierarchy":{"paths":[["doc:\/\/AWSAppflow\/documentation\/AWSAppflow","doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/DeleteConnectorProfileOutputError"]]},"references":{"doc://AWSAppflow/documentation/AWSAppflow/ConflictException":{"role":"symbol","title":"ConflictException","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ConflictException"}],"abstract":[{"type":"text","text":"There was a conflict when processing the request (for example, a flow with the given name already exists within the account. Check for conflicting resource names and try again."}],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/ConflictException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ConflictException"}],"url":"\/documentation\/awsappflow\/conflictexception"},"doc://AWSAppflow/documentation/AWSAppflow":{"role":"collection","title":"AWSAppflow","abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow","kind":"symbol","type":"topic","url":"\/documentation\/awsappflow"},"doc://AWSAppflow/documentation/AWSAppflow/DeleteConnectorProfileOutputError":{"role":"symbol","title":"DeleteConnectorProfileOutputError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"DeleteConnectorProfileOutputError"}],"abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/DeleteConnectorProfileOutputError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"DeleteConnectorProfileOutputError"}],"url":"\/documentation\/awsappflow\/deleteconnectorprofileoutputerror"},"doc://AWSAppflow/documentation/AWSAppflow/DeleteConnectorProfileOutputError/conflictException(_:)":{"role":"symbol","title":"DeleteConnectorProfileOutputError.conflictException(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"conflictException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ConflictException","preciseIdentifier":"s:10AWSAppflow17ConflictExceptionV"},{"kind":"text","text":")"}],"abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/DeleteConnectorProfileOutputError\/conflictException(_:)","kind":"symbol","type":"topic","url":"\/documentation\/awsappflow\/deleteconnectorprofileoutputerror\/conflictexception(_:)"}}}